src/mono/mono/mini/llvm-intrinsics.c

```diff
@@ -0,0 +1,534 @@
+  [NI_Vector128_get_Zero] = {EMIT_MANUAL},
+  [NI_Vector128_GetElement] = {EMIT_MANUAL},
+  [NI_Vector128_GetLower] = {EMIT_MANUAL},
+  [NI_Vector128_GetUpper] = {EMIT_MANUAL},
+  [NI_Vector128_GreaterThan] = {EMIT_MANUAL},
+  [NI_Vector128_GreaterThanAll] = {EMIT_MANUAL},
+  [NI_Vector128_GreaterThanAny] = {EMIT_MANUAL},
+  [NI_Vector128_GreaterThanOrEqual] = {EMIT_MANUAL},
+  [NI_Vector128_GreaterThanOrEqualAll] = {EMIT_MANUAL},
+  [NI_Vector128_GreaterThanOrEqualAny] = {EMIT_MANUAL},
+  [NI_Vector128_LessThan] = {EMIT_MANUAL},
+  [NI_Vector128_LessThanAll] = {EMIT_MANUAL},
+  [NI_Vector128_LessThanAny] = {EMIT_MANUAL},
+  [NI_Vector128_LessThanOrEqual] = {EMIT_MANUAL},
+  [NI_Vector128_LessThanOrEqualAll] = {EMIT_MANUAL},
+  [NI_Vector128_LessThanOrEqualAny] = {EMIT_MANUAL},
+  [NI_Vector128_Load] = {EMIT_MANUAL},
+  [NI_Vector128_LoadAligned] = {EMIT_MANUAL},
+  [NI_Vector128_LoadAlignedNonTemporal] = {EMIT_MANUAL},
+  [NI_Vector128_LoadUnsafe] = {EMIT_MANUAL},
+  [NI_Vector128_Max] = {EMIT_MANUAL},
+  [NI_Vector128_Min] = {EMIT_MANUAL},
+  [NI_Vector128_Multiply] = {EMIT_MANUAL},
+  [NI_Vector128_Narrow] = {EMIT_MANUAL},
+  [NI_Vector128_Negate] = {EMIT_MANUAL},
+  [NI_Vector128_OnesComplement] = {EMIT_MANUAL},
+  [NI_Vector128_op_Addition] = {EMIT_MANUAL},
+  [NI_Vector128_op_BitwiseAnd] = {EMIT_MANUAL},
+  [NI_Vector128_op_BitwiseOr] = {EMIT_MANUAL},
+  [NI_Vector128_op_Division] = {EMIT_MANUAL},
+  [NI_Vector128_op_Equality] = {EMIT_MANUAL},
+  [NI_Vector128_op_ExclusiveOr] = {EMIT_MANUAL},
+  [NI_Vector128_op_Inequality] = {EMIT_MANUAL},
+  [NI_Vector128_op_LeftShift] = {EMIT_MANUAL},
+  [NI_Vector128_op_RightShift] = {EMIT_MANUAL},
+  [NI_Vector128_op_UnsignedRightShift] = {EMIT_MANUAL},
+  [NI_Vector128_op_Multiply] = {EMIT_MANUAL},
+  [NI_Vector128_op_OnesComplement] = {EMIT_MANUAL},
+  [NI_Vector128_op_Subtraction] = {EMIT_MANUAL},
+  [NI_Vector128_op_UnaryNegation] = {EMIT_MANUAL},
+  [NI_Vector128_op_UnaryPlus] = {EMIT_MANUAL},
+  [NI_Vector128_ShiftLeft] = {EMIT_MANUAL},
+  [NI_Vector128_ShiftRightArithmetic] = {EMIT_MANUAL},
+  [NI_Vector128_ShiftRightLogical] = {EMIT_MANUAL},
+  [NI_Vector128_Shuffle] = {EMIT_MANUAL},
+  [NI_Vector128_Sqrt] = {EMIT_MANUAL},
+  [NI_Vector128_Store] = {EMIT_MANUAL},
+  [NI_Vector128_StoreAligned] = {EMIT_MANUAL},
+  [NI_Vector128_StoreAlignedNonTemporal] = {EMIT_MANUAL},
+  [NI_Vector128_StoreUnsafe] = {EMIT_MANUAL},
+  [NI_Vector128_Subtract] = {EMIT_MANUAL},
+  [NI_Vector128_Sum] = {EMIT_MANUAL},
+  [NI_Vector128_ToScalar] = {EMIT_MANUAL},
+  [NI_Vector128_WidenLower] = {EMIT_MANUAL},
+  [NI_Vector128_WidenUpper] = {EMIT_MANUAL},
+  [NI_Vector128_WithElement] = {EMIT_MANUAL},
+  [NI_Vector128_WithLower] = {EMIT_MANUAL},
+  [NI_Vector128_WithUpper] = {EMIT_MANUAL},
+  [NI_Vector128_Xor] = {EMIT_MANUAL},
+
+  // ...---===[ Arm64 Advanced SIMD ]===---...
+  [NI_AdvSimd_Abs] = {OP_XOP_OVR_X_X, INTRINS_AARCH64_ADV_SIMD_ABS, NONE, NONE, OP_XOP_OVR_X_X, INTRINS_AARCH64_ADV_SIMD_FABS},
```

Comment:
For architecture specific intrinsics like `AdvSimd`, I think the named intrinsics Mono receives will be something like `INS_abs`. We need a mapping from `INS_abs` to `NI_AdvSimd_Abs`. That's the original thought.